app/api.rb

```diff
@@ -1,23 +1,26 @@
 require 'grape'
-require 'sequel'
-require 'pg'
-require_relative 'services/json_web_token'
-
-DB = Sequel.connect(
-  adapter: 'postgres',
-  host: 'db',
-  database: 'conduit',
-  user: 'postgres',
-  password: 'abc'
-)
-
 require_relative 'models/user'
+require_relative 'decorators/user'
 
 module Conduit
   class API < Grape::API
     prefix :api
     format :json
 
+    helpers do
```

Comment:
This Controller file now uses helper methods and models/decorators/services to do the actions that it did before itself.